tests/partitioner/test_histogram.py

```diff
@@ -51,3 +53,54 @@ def test_column_names():
     expected = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8]
     npt.assert_array_equal(result, expected)
     assert (result == expected).all()
+
```

Comment:
should probably have tests for the two value errors that can be raised (when histogram isn't right size and when it exceeds threshold)

Reply:
Abso-toot-ly.